runtime/bindings/python/tests/test_ngraph/test_preprocess.py

```diff
@@ -398,21 +343,11 @@ def test_ngraph_preprocess_model():
     def custom_preprocess(output: Output):
         return ops.abs(output)
 
-    function = PrePostProcessor(function) \
-        .input(InputInfo(1)
-               .preprocess(PreProcessSteps()
-                           .convert_element_type(Type.f32)
-                           .scale(0.5)
-                           )
-               ) \
-        .input(InputInfo(0)
-               .preprocess(PreProcessSteps()
-                           .convert_element_type(Type.f32)
-                           .mean(5.))) \
-        .output(OutputInfo(0)
-                .postprocess(PostProcessSteps()
-                             .custom(custom_preprocess))) \
-        .build()
+    p = PrePostProcessor(function)
+    p.input(1).preprocess().convert_element_type(Type.f32).scale(0.5)
```

Comment:
There is test for input(index), can you please also add tests for `input(tensor_name)`? Same is for `output()`

Reply:
Added.